frontend/front/src/pages/Admin/nav/Nav.js

```diff
@@ -11,11 +11,14 @@ const Nav = () => {
     <Box sx={{ flexGrow: 1 }}>
       <AppBar position="static">
         <Toolbar>
-          <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
+          <Box sx={{ flexGrow: 1 }}>
             <Button color="inherit" component={Link} to="">
-              <Typography variant="h6">ADMIN</Typography>
+              <Typography variant="h6">DASHBOARD</Typography>
             </Button>
-          </Typography>
+            <Button color="inherit" component={Link} to="/surveyor/dashboard">
+              <Typography variant="h7">SURVEYOR MODE</Typography>
```

Comment:
Admins can now click "Surveyor Mode" in the nav to be treated like a surveyor

Reply:
What Surveyor does it mimic?

Reply:
It just behaves as if the admin user was a surveyor user